src/otx/v2/adapters/torch/mmengine/mmdet/modules/evaluation/evaluator.py

```diff
@@ -0,0 +1,469 @@
+"""Evaluator of OTX Detection."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import copy
+import multiprocessing as mp
+from collections import OrderedDict
+from typing import TYPE_CHECKING, Any, Sequence
+
+if TYPE_CHECKING:
+    import logging
+
+import numpy as np
+import pycocotools.mask as mask_util
+import torch
+from mmdet.evaluation.functional.class_names import get_classes
+from mmdet.evaluation.functional.mean_ap import average_precision, eval_map
+from mmdet.evaluation.metrics import VOCMetric
+from mmdet.registry import METRICS
+from mmdet.structures.bbox import bbox_overlaps
+from mmdet.structures.mask import encode_mask_results
+from mmdet.structures.mask.structures import BitmapMasks, PolygonMasks
+from mmengine.logging import MMLogger, print_log
+from mmengine.utils.misc import is_str
+from terminaltables import AsciiTable
+
+from otx.v2.api.entities.label import Domain
+
+
+class ValueDifferenceError(Exception):
+    """Custom Exception when two values are different when they should be same."""
+
+    def __init__(self, value1: int | tuple[int], value2: int | tuple[int]) -> None:
+        """Initialize Exception.
+
+        Args:
+            value1: (int | tuple[int])
+            value2: (int | tuple[int])
+        """
+        super().__init__(f"{value1} and {value2} should be same.")
+
+
+def print_map_summary(  # pylint: disable=too-many-locals,too-many-branches
```

Comment:
We don't need pylint disable anymore
```suggestion
def print_map_summary(
```